src/test/java/core/basesyntax/service/impl/FileReaderServiceImplTest.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.exception.InvalidValueExeption;
+import core.basesyntax.service.CsvFileReaderService;
+import java.util.Collections;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class FileReaderServiceImplTest {
+    private static CsvFileReaderService fileReaderService;
+    private static final String VALID_FILE_PATH = "src/test/resources/TestData.csv";
+    private static final String INVALID_FILE_PATH = "src/resources/NonexistentFile.csv";
+    private static final String INVALID_FORMAT_FILE_PATH
+            = "src/test/resources/InvalidFormatData.csv";
+    private static final String EMPTY_FILE_PATH = "src/test/resources/EmptyData.csv";
+
+    @BeforeAll
+    static void setUp() {
+        fileReaderService = new FileReaderServiceImpl();
+    }
+
+    @Test
+    void readFile_readValidFile_ok() {

```

Comment:
```suggestion
    public void readFile_readValidFile_ok() {

```

Make your test signatures consistent.